srcs/layouts/latn_qwerty_kk.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (Qazaqşa)" script="latin">
+  <modmap>
+    <shift a="i" b="İ" />
+    <shift a="ı" b="I" />
+  </modmap>
+  <row>
+    <key shift="0.5" key0="ä"/>
+    <key key0="ñ"/>
+    <key key0="ğ"/>
+    <key key0="ç"/>
+    <key key0="ū"/>
+    <key key0="ü"/>
+    <key key0="ı"/>
+    <key key0="ö"/>
+    <key key0="ş"/>
+  </row>
```

Comment:
These letters are present on the layout at two position at the same time: `ä, ç, ñ, ö, ü, ğ, ı, ş, ū`. It seems that the top row could be entirely removed.

Reply:
Removed them from swipe keys